databricks/koalas/indexes.py

```diff
@@ -261,6 +276,21 @@ def any(self, *args, **kwargs):
     def all(self, *args, **kwargs):
         raise TypeError("cannot perform all with this index type: MultiIndex")
 
+    @property
+    def size(self) -> int:
```

Comment:
DataFrame and Series have it in `general.py`. So, it we add it to `base.py`, then it's overlapped with Series's .. so I had to add it in each.

Reply:
I don't think we need this here except for the example in the doctest, need only in `Index` since `MultiIndex` extends `Index`.

Reply:
Oops, yes. Argh, I rushed.